src/app/navigation/Home.screen.tsx

```diff
@@ -58,4 +70,9 @@ const styles = StyleSheet.create({
     fontSize: 16,
     flexShrink: 1,
   },
+  image: {
+    width: 50,
+    height: 75,
+    marginRight: 10,
+  },
 });
```

Comment:
we can type style so when we define a style for View we do not put style of Image for instance ... etc